src/App/Webhook/PdkWebhookManager.php

```diff
@@ -123,8 +117,17 @@ protected function resolveHook(string $event): HookInterface
      */
     private function getHooks(Request $request): array
     {
-        $body = json_decode($request->getContent(), true);
+        $headers = $request->headers->all();
+        $body    = json_decode($request->getContent(), true);
+
+        if (! isset($headers['x-myparcel-hook'])) {
+            return $body['data']['hooks'] ?? [];
```

Comment:
je doet nu 2x `$body['data']['hooks']`, dat hoeft niet, graag in een variabele zetten. en de onderste is niet null safe